force-app/main/default/lwc/bulkServiceDeliveryUI/bulkServiceDeliveryUI.js

```diff
@@ -195,4 +195,11 @@ export default class BulkServiceDeliveryUI extends NavigationMixin(LightningElem
     get isDoneDisabled() {
         return Object.keys(this.errors).length > 0;
     }
+
+    get doneTitleLabel() {
+        if (this.isDoneDisabled) {
```

Comment:
What do you think about renaming the variable isDoneDisabled to something like hasErrors?